README.md

```diff
@@ -27,6 +27,18 @@ dependencies {
 
 For more information, you can refer to the [Using with ktlint](https://twitter.github.io/compose-rules/#using-the-custom-ruleset-with-ktlint) documentation.
 
+## Using with kotlinter
```

Comment:
Can you add this to docs/index.md too? In there you could also add a link to the kotlinter project as well